### PR TITLE
[Fix] Fix tests to run without breaking when firebase spec key isn't present

### PR DESCRIPTION
**Pull request by benjaminkostiuk**
## Overview

Implements mock `firebase_admin` module to use with tests that need to import from firebase. Add it to the tests so that they pass on CI.

Tests should now be fixed once @dizona updates his tests to mock the firebase module.

This change is a

- [x] Bug fix
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that will change exisiting functionality)
- [ ] Documentation update

## Motivation/Links
Tests need to pass on CI as well as local.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+from firebase_admin import credentials, initialize_app
 
+
+# Initializing Firestore database
+# Can import firestore_db to utilize database
+def initialize_firebase():
+    initialize_app(credentials.Certificate("serviceAccountKey.json"))
+
+
+initialize_firebase()
```
Comment (ruler225) [inline manage.py:12]: Wait so why is this enclosed in a function and not just run directly?

**Review by ruler225 — APPROVED** [review]
Looks fine, though CI still seems to be failing

File: manage.py (hunk 1 above)
Comment (benjaminkostiuk) [reply manage.py:12]: Functions are generally easier to test. And I might want to import it in a end-to-end test.

Comment (benjaminkostiuk) [reply manage.py:12]: But for now it would be the same you're right.

**Comment by benjaminkostiuk** [comment]
@ruler225 It's failing on Adrian's tests. Once he changes them to mock firebase it should work.

**Comment by ruler225** [comment]
@benjaminkostiuk I saw it failing on one of the events tests too

**Comment by benjaminkostiuk** [comment]
> @benjaminkostiuk I saw it failing on one of the events tests too

Gonna make me cry 😭

**Comment by benjaminkostiuk** [comment]
@ruler225 I forgot to update the event tests I think when I changed the decorator for rewards. I'll handle it when I write tests for rewards this weekend.

**Pull request merged**